ansible/build-archive-opt-custom.yml

```diff
@@ -0,0 +1,21 @@
+- name: Check builder host
```

Comment:
I suggest to make the build playbook similar to the build-archive-monitoring playbook, where all steps are performed on a clean zone, which simulates the compute node environment.